### PR TITLE
New wait flag for install/upgrade

### PR DESCRIPTION
**Pull request by mhmxs**
This new flag helps automation nerds to wait until cluster is ready.

### CODE REVIEW

File: pkg/installer/upgrade.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func Upgrade(uninstallConfig *apiv1.KubectlStorageOSConfig, installConfig *apiv1
 	time.Sleep(30 * time.Second)
 
 	// install new storageos operator and cluster
-	err = installer.installStorageOS(installConfig)
+	err = installer.Install(installConfig)
```
Comment (mhmxs) [inline pkg/installer/upgrade.go:58]: I think better to not be clever here and call the installer as it is. If include etcd is false - as it is - it will skip installation of it so will do the same :D

File: cmd/plugin/cli/install.go [hunk 2]
```diff
@@ -30,6 +30,7 @@ func InstallCmd() *cobra.Command {
 			return nil
 		},
 	}
+	cmd.Flags().Bool(installer.WaitFlag, false, "wait for operation finish")
```
Comment (nolancon) [inline cmd/plugin/cli/install.go:33]: Nit: I think this should be a bit more descriptive, maybe:
 `wait for storagos cluster to enter running phase` ?

File: cmd/plugin/cli/upgrade.go [hunk 3]
```diff
@@ -41,6 +41,7 @@ func UpgradeCmd() *cobra.Command {
 			return nil
 		},
 	}
+	cmd.Flags().Bool(installer.WaitFlag, false, "wait for operation finish")
```
Comment (nolancon) [inline cmd/plugin/cli/upgrade.go:44]: Nit: I think this should be a bit more descriptive, maybe:
 `wait for storagos cluster to enter running phase` ?

File: pkg/installer/installer.go [hunk 4]
```diff
@@ -40,8 +41,9 @@ const (
 
 	// config file fields - contain path delimiters for plugin interpretation of config manifest
 	SkipNamespaceDeletionConfig   = "spec.skipNmespaceDeletion"
```
Comment (nolancon) [inline pkg/installer/installer.go:43]: also just noticed this - not part of this PR - but should this be `spec.skipNamespaceDeletion` ?

**Review by nolancon — COMMENTED** [review]
couple of nits, otherwise LGTM

**Review by nolancon — APPROVED** [review]
